e2e/testcases/multiversion_test.go

```diff
@@ -58,6 +58,22 @@ func TestMultipleVersions_CustomResourceV1(t *testing.T) {
 		nt.T.Fatal(err)
 	}
 
+	nt.T.Cleanup(func() {
```

Comment:
This cleanup shouldn't be necessary, since everything is managed by the RootRepo. The test scaffolding will handle cleanup for anything managed by the RootRepo in [Reset](https://github.com/GoogleContainerTools/kpt-config-sync/blob/main/e2e/nomostest/reset.go#L83).